src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/form/common/edit-form.js

```diff
@@ -104,6 +106,13 @@ define([
 
       this.getRoot().trigger('pim_enrich:form:render:after');
       this.getRoot().trigger('pim_enrich:form:extension:render:after');
+
+      if (this.model && this.model.attributes) {
+        analytics.track('form:edit:opened', {
+          code: this.code,
+          attributes: this.model.attributes,
+        });
+      }
```

Comment:
We should not add business rules when it not required.
The control of the attributes should be handle where we listen the event

```suggestion
        analytics.track('form:edit:opened', {
          code: this.code,
          model: this.model,
        });
```